src/microservices/users/src/Api/Handlers/UserHandlerV1.cs

```diff
@@ -0,0 +1,93 @@
+namespace Api
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Linq;
+    using System.Threading.Tasks;
+    using Api.Protos;
+    using Google.Protobuf.WellKnownTypes;
+    using Grpc.Core;
+
+    public class UserHandlerV1 : Users.UsersBase
```

Comment:
We can rename that to `UserHandler` only.

Reply:
It's a good point.

We must consider how we want to handle breaking changes in gRPC services. One of the options is versioning handlers and package names. Here [link](Versioning gRPC services) with more details on that approach.

But we must review other alternatives as well.